tests/unit_tests/html.fixtures.js

```diff
@@ -535,8 +574,12 @@ html.proxies_non_match = `
 // ============================
 // Signature
 // ============================
+// I think signature pages won't have sought var node on mobile
```

Comment:
TODO: I need to remove this comment. It's solved as long as the interview developer does the thing that needs doing.

On the other hand, maybe we need to handle situations where the sought var is not present on the page to allow for more simple interviews to still be tested.

Reply:
It seems I've already handled this. See the `.addToReport()` a bit further on.

Reply:
Comment adjusted.